Doc/reference/compound_stmts.rst

```diff
@@ -90,8 +90,8 @@ The :keyword:`!if` statement
 The :keyword:`if` statement is used for conditional execution:
 
 .. productionlist::
-   if_stmt: "if" `expression` ":" `suite`
-          : ("elif" `expression` ":" `suite`)*
+   if_stmt: "if" `assignment_expression` ":" `suite`
+          : ("elif" `assignment_expression` ":" `suite`)*
           : ["else" ":" `suite`]
 
 It selects exactly one of the suites by evaluating the expressions one by one
```

Comment:
Do you think "expression" here needs updating too? It becomes a bit ugly that way, and in context it doesn't seem ambiguous.

Reply:
(I thought more about it and I don't think it needs updating, so I'll land now.)